package.json

```diff
@@ -107,7 +107,7 @@
     "react-swipeable-views": "^0.14.0",
     "react-textarea-autosize": "^8.4.0",
     "react-toggle": "^4.1.3",
-    "redis": "^4.0.6 <4.1.0",
+    "redis": "^4.0.6 <4.7.0",
```

Comment:
```suggestion
    "redis": "^4.6.4",
```

Reply:
Updated the comment since I noticed 4.6.5 is out now https://github.com/nschonni/mastodon/pull/93